js/app.js

```diff
@@ -1,4 +1,25 @@
 import '@dosomething/forge';
 import '../sass/app.scss';
 
-// JS goes here.
+import React from 'react';
+import ReactDom from 'react-dom';
+import 'babel-polyfill';
+
+import Leaderboard from './components/Leaderboard'
+
+// Helper to decide if the DOM is ready.
+export function ready(fn) {
+    if (document.readyState !== 'loading') {
+        fn();
+    } else {
+        document.addEventListener('DOMContentLoaded', fn);
+    }
+}
+
+// Actually add the component to the page when it is ready.
+ready(() => {
+	ReactDom.render(
+		<Leaderboard />,
+		document.getElementById('leaderboard')
+	);
+})
```

Comment:
new line 😬

Reply:
ah! I thought I had that package to do that for me! Thanks :)

Reply:
I added these new lines but for some reason not showing up in the PR 🤔